File: 1. Iniciante/URI1046.py
x,y = input().split(" ")

HI = int(x)
HF = int(y)
total = 0

if HI > HF:
    for a in range(HI + 1, 25):
        total += 1

        if a == 24:
            for b in range(1, HF+1):
                total += 1
elif HI < HF:
    total = HF - HI
else:
    total = 24
print("O JOGO DUROU %d HORA(S)" %total)

File: 1. Iniciante/URI1046.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    int HI =0, HF=0, total = 0;

    cin >> HI >> HF;

    if (HI > HF) {
        for(int x = HI + 1; x <= 24; x++) {
            total += 1;
            if (x == 24){
                for (int x = 1; x <= HF; x++) {
                    total += 1;
                }
            }
        }
    } else if( HI < HF){
        total = HF - HI;
    } else {
        total = 24;
    }

    printf("O JOGO DUROU %d HORA(S)\n", total);

    return 0;
}
